app/texters/rainy_day_texter.rb

```diff
@@ -11,9 +11,10 @@ def self.send_admin_text(scheduled_time)
   private
 
   def self.rainy_day_text(garden, chance)
+
     {
       from: TwillioAccountPhoneNumber,
-      to: "+1#{garden.user.telephone}",
+      to: "+1#{garden.users.first.telephone}",
```

Comment:
I think we will want to re-visit how we set up text messages... Maybe per garden?